SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py

```diff
@@ -99,6 +99,12 @@ def customise_DQM(process):
     #process.dqmoffline_step.remove(process.jetMETAnalyzer)
     # Turn off flag of gangedME11a
     process.l1tCsctf.gangedME11a = cms.untracked.bool(False)
+    # Turn off "low bias voltage" region in HCAL noise filters
+    if hasattr(process,'HBHENoiseFilterResultProducer'):
+        process.HBHENoiseFilterResultProducer.IgnoreTS4TS5ifJetInLowBVRegion = cms.bool(False)
+    # Use a different HBHE noise filter flag in caloMetDQMAnalyzer
+    if hasattr(process,'caloMetDQMAnalyzer'):
```

Comment:
[it looks like my comment in https://github.com/cms-sw/cmssw/pull/8685#commitcomment-10742971 went unnoticed, since I commented in the commit itself[
this is not enough: it only modifies this instance, but not the clones.
The same change should be applied to pfMetDQMAnalyzer and pfMetT1DQMAnalyzer